cmd/cli/config/set.go

```diff
@@ -90,8 +89,8 @@ This command supports two input formats for setting configuration values:
 	return setCmd
 }
 
-func setConfig(cfgFilePath, key string, value ...string) error {
-	log.Info().Msgf("Writing config to %s", cfgFilePath)
+func setConfig(cmd *cobra.Command, cfgFilePath, key string, value ...string) error {
+	cmd.Printf("Writing config to %s", cfgFilePath)
```

Comment:
We should write this line right before `if err := v.WriteConfigAs(cfgFilePath); err != nil {` to ensure we only print it when we write the file.

Reply:
is says `Writing` and not `Wrote`. Useful to know when it is intending to write so failures can make more sense